packages/@aws-cdk/aws-chatbot/test/integ.chatbot-from-arn.js.snapshot/ChatbotFromArnInteg.template.json

```diff
@@ -0,0 +1,73 @@
+{
+ "Resources": {
+  "MySlackChannelConfigurationRole1D3F23AE": {
+   "Type": "AWS::IAM::Role",
+   "Properties": {
+    "AssumeRolePolicyDocument": {
+     "Statement": [
+      {
+       "Action": "sts:AssumeRole",
+       "Effect": "Allow",
+       "Principal": {
+        "Service": "chatbot.amazonaws.com"
+       }
+      }
+     ],
+     "Version": "2012-10-17"
+    }
+   }
+  },
+  "MySlackChannelA8E0B56C": {
+   "Type": "AWS::Chatbot::SlackChannelConfiguration",
+   "Properties": {
+    "ConfigurationName": "test-channel",
+    "IamRoleArn": {
+     "Fn::GetAtt": [
+      "MySlackChannelConfigurationRole1D3F23AE",
+      "Arn"
+     ]
+    },
+    "SlackChannelId": "C0187JABUE9",
+    "SlackWorkspaceId": "T49239U4W",
+    "LoggingLevel": "NONE"
+   }
+  },
+  "MyTopic86869434": {
```

Comment:
First, I'm not an expert.
It looks like the snapshots that the test runs against is from this commit [ed8373850e](https://github.com/bzbislawski/aws-cdk/commit/ed8373850e20ee6398d11ba93898a5d6422b2496) where the topic(s) is not the same as for the current state which causes the __integ.chatbot-from-arn__ test to fail.

Can't we just re-generate the snapshot template by remove `test/integ.chatbot-from-arn.js.snapshot/` and re-run the test to generate the new snapshot?
This is the base snapshot so it should be OK right?

I "fixed" this by:
1. Delete `test/integ.chatbot-from-arn.js.snapshot/`
2. `yarn test` -> fail
3. Rename the snapshot folder that was generated from `test/cdk-integ.out.integ.chatbot-from-arn.js.snapshot` to `itest/nteg.chatbot-from-arn.js.snapshot`
4. `yarn test` -> success